doc/classes/EditorSettings.xml

```diff
@@ -941,6 +941,9 @@
 		<member name="interface/inspector/auto_unfold_foreign_scenes" type="bool" setter="" getter="">
 			If [code]true[/code], automatically expands property groups in the Inspector dock when opening a scene that hasn't been opened previously. If [code]false[/code], all groups remain collapsed by default.
 		</member>
+		<member name="interface/inspector/dragging_unfold_wait_seconds" type="float" setter="" getter="">
+			During a drag-and-drop, this is how long to wait over a section before the section unfolds.
```

Comment:
This is a bit misleading, because it affects all Trees in the editor.

Reply:
I'll change it to mention that it also affects other editor interfaces that unfold on drag-n-drop. Or should this have a new section?

Reply:
It can go into `interface/editor` probably.